### PR TITLE
WA for lvmo snapshot and clone  for 2 feature_blocker and label change

### PR DESCRIPTION
**Pull request by shyRozen**
TL;DR

- from 4.11.0-99 lvmo pod in crashloopback. There is an automation workaround and it is applied from 99-110 where it is solved.
- from 4.11.0-105 labels where change.
- from 4.11.0.105 restores of snapshot and clone are empty till a build with the fix will be available
Need to continue deployment for latest versions for other features than snap/clone and for automation effort of snap/clone need to be able to deploy older versions than 4.11.0-105
Till latest will be stable this is the only solution to keep on working. Even PR validations will not work without this fix. not so tldr :)

In 4.11-105 lvmo pods labels were changed and [#6079 ](https://github.com/red-hat-storage/ocs-ci/pull/6079) was merged. Now that we have a blocker [BZ-2103818](https://bugzilla.redhat.com/show_bug.cgi?id=2103818) that is applicable from 4.11-105 and above we and also have  [BZ-2101139](https://bugzilla.redhat.com/show_bug.cgi?id=2101139) which need WA from version 4.11-99-110 also we are stuck with PR validation and tests. This PR is intended to allow install all versions to allow other features to work on latest and snapshot and clone for earlier builds. Also PR validation not work without this fix.
Signed-off-by: Shay Rozen <shay.rozen@gmail.com>

### CODE REVIEW

File: ocs_ci/deployment/deployment.py [hunk 1]
```diff
@@ -1240,6 +1241,13 @@ def deploy_lvmo(self):
         lvmo_version_without_period = lvmo_version.replace(".", "")
         label_version = constants.LVMO_POD_LABEL
         create_catalog_source()
+        lvm_full_version = get_lvm_full_version()
+        major, minor = lvm_full_version.split("-")[1]
+        if int(minor) > 105 and major == "4.11":
+            lvmo_version_without_period = "411"
+        elif int(minor) < 105 and major == "4.11":
+            lvmo_version_without_period = "411-old"
```
[the remark this answers is not in the document]
Comment (shyRozen) [reply ocs_ci/deployment/deployment.py:1249]: We need it for a workaround to keep developing automation. Above 4.11-105 the label change is there but the feature doesn't work. Below 4.11-105 the feature is working but the labels are old. This enable to choose the right key for the right labels so we can deploy for now below 4.11.-105 to continue some automation effort.

**Review by Avilir — APPROVED** [review]
LGTM

File: ocs_ci/deployment/deployment.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (shyRozen) [reply ocs_ci/deployment/deployment.py:1249]: Test with the latest when it can be tested and not broken. I want also the option to deploy older builds - now because they are the only one that works and maybe in the future for comparison.

**Comment by shyRozen** [comment]
Also tested locally 3 versions to fulfill all conditions.

File: ocs_ci/deployment/deployment.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ebenahar) [reply ocs_ci/deployment/deployment.py]: please open a workaround issue that will track the removal of these conditions

**Review by ebenahar — APPROVED** [review]

File: ocs_ci/deployment/deployment.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (shyRozen) [reply ocs_ci/deployment/deployment.py]: update #6074
